### ISSUE TITLE
[feature request] Complete Project Code 

### ISSUE THREAD

**Issue by PlumedSerpent** [D-X-Y/landmark-detection#33]
## Which project are you using?
 TS3

## Issue description

To my understanding , the files under models DIR are not all the source codes, right? Would you release the dataloader part and train function days later?

**Comment by D-X-Y** [comment]
Thanks for your interest. I'm re-organizing the TS3 project codes, which are a little bit messy. However, I'm facing some imminent deadlines, and thus it could take much time to release the full codes. Thanks for your patience!

**Comment by PlumedSerpent** [comment]
@D-X-Y I see. That's OK. I'll try to re-implement according to your paper. Can you provide some "training logs"  to guide me?

**Comment by D-X-Y** [comment]
Thanks for your understanding. I'll try to find logs and give back to you.

**Comment by xuxixia** [comment]
> @D-X-Y I see. That's OK. I'll try to re-implement according to your paper. Can you provide some "training logs" to guide me?

hello, friend. I also plan to re-implement this code as well. Are u finished it? Would you like to talk about some details together ?

**Comment by D-X-Y** [comment]
Hi @xuxixia , thanks for your interest. If you have any questions, it's my pleasure to answer.

**Comment by likui01** [comment]
Hi,
@D-X-Y , will you be uploading the training code?
If yes, when ?
thanks

**Comment by ravit-cohen-segev** [comment]
Hi, Can you elaborate on the student's output dim/ teachers' input dim? what shape should it be?

**Comment by ravit-cohen-segev** [comment]
I'd also like the training logs
